TF_Service_dotNet/TouchFreeTests/VirtualScreenTests.cs

```diff
@@ -0,0 +1,144 @@
+﻿using System.Numerics;
+using NUnit.Framework;
+using TouchFreeTests.TestImplementations;
+using Ultraleap.TouchFree.Library;
+using Ultraleap.TouchFree.Library.Configuration;
+
+namespace TouchFreeTests
+{
+    public class VirtualScreenTests
+    {
+        [Test]
+        public void Constructor_ValidInputs_ReturnsInstance()
+        {
+            //Given
+            IConfigManager configManager = new TestConfigManager();
+
+            //When
+            VirtualScreen virtualScreen = new VirtualScreen(1080, 1920, 0.4f, 0, configManager);
+
+            //Then
+            Assert.AreEqual(1080, virtualScreen.Width_VirtualPx);
+            Assert.AreEqual(1920, virtualScreen.Height_VirtualPx);
+            Assert.AreEqual(0.4f, virtualScreen.Height_PhysicalMeters);
+            Assert.AreEqual(0.225f, virtualScreen.Width_PhysicalMeters, 0.001);
+            Assert.AreEqual(0, virtualScreen.AngleOfPhysicalScreen_Degrees);
+            Assert.AreEqual(0, virtualScreen.PhysicalScreenPlane.Normal.X);
+            Assert.AreEqual(0, virtualScreen.PhysicalScreenPlane.Normal.Y);
+            Assert.AreEqual(1, virtualScreen.PhysicalScreenPlane.Normal.Z);
+        }
+
+        private int ScreenWidthInPixels = 1080;
```

Comment:
Although it's good to use realistic screen dimensions I wonder if it would still be valid to use a screen size such as 1000x1000 to make the comparisons in the test cases easier to read. I had to go to my calculator to check the values.

Not suggesting we have to change it now though, the values all look correct